docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform.md

```diff
@@ -0,0 +1,99 @@
+# DigitalOcean App Platform
+
+This is a step-by-step guide for deploying a Strapi project to [DigitalOcean's App Platform](https://digitalocean.com). App Platform is DigitalOcean's Platform as a Service (PaaS) that will handle deploying, networking, SSL, and more for your app. It is the easiest way to deploy Strapi to DigitalOcean.
+
+Databases can be created using DigitalOcean's [Managed Databases](https://www.digitalocean.com/products/managed-databases/).
+
+Prior to starting this guide, you should have created a [Strapi project](/developer-docs/latest/getting-started/quick-start.md). And have read through the [configuration](/developer-docs/latest/setup-deployment-guides/deployment.md#application-configuration) section.
+
+::: tip
+Strapi does have a [One-Click](/developer-docs/latest/setup-deployment-guides/installation/digitalocean-one-click.md) deployment option for DigitalOcean and can also be deployed to DigitalOcean Droplets.
+:::
+
+### DigitalOcean Install Requirements
+
+- If you don't have a DigitalOcean account you will need to create one, you can use [this referral link](https://try.digitalocean.com/strapi/) to get \$100 of free credits!
+
+## Configure Your Strapi Project for Deployment
+
+To deploy your Strapi app, you will need to create a database configuration file. You will be using PostgreSQL for this example but you are able to connect to any of the [databases](https://docs.digitalocean.com/products/databases/) provided by DigitalOcean and [supported by Strapi](/developer-docs/latest/setup-deployment-guides/configurations.html#database).
+
+You will configure a database for production. With the setup below, you will only need to set **one environment variable** for the `DATABASE_URL` to connect to your PostgreSQL database. Add the following to `config/env/production/database.js`:
+
+```javascript
+const parse = require('pg-connection-string').parse;
+const config = parse(process.env.DATABASE_URL);
+
+module.exports = () => ({
+  defaultConnection: 'default',
+  connections: {
+    default: {
+      connector: 'bookshelf',
+      settings: {
+        client: 'postgres',
+        host: config.host,
+        port: config.port,
+        database: config.database,
+        username: config.user,
+        password: config.password,
+        ssl: {
+          rejectUnauthorized: false,
+        },
+      },
+      options: {},
+    },
+  },
+});
+```
+
+Your application is now ready to deploy to DigitalOcean App Platform.
+
+## Deploying Strapi to DigitalOcean App Platform
+
+App Platform lets you deploy your application directly from a [GitHub](https://github.com) repo. [Gitlab](https://gitlab.com) is also supported.
+
+### Step 1. Log in to your [DigitalOcean account](https://cloud.digitalocean.com/login).
+
```

Comment:
Not required but do you want to add some screenshots? We can create a new assets folder in https://github.com/strapi/documentation/tree/main/docs/developer-docs/latest/assets for deployment assets. I believe we are planning to at a later date but it's up to you if you want to do it now.

Reply:
theres some movement happening in the ui so i think screenshots may get outdated quickly. i can revisit after a month or so

Reply:
No problem, we will table it for now. Interested to see the UI changes 👀